backend/docker-compose-testnet.yaml

```diff
@@ -0,0 +1,191 @@
+x-common-environment: &common-environment
```

Comment:
Could we get this out of the backend and include the frontend as well?

Reply:
I don't think there's any overlap in the environment for the frontend, and yes I will bring the frontend in.